File: src/services/schema-export/tests/models/all/java/ReverseType.java
// Please note : @LinkingObjects and default values are not represented in the schema and thus will not be part of the generated models
package your.package.name.here;

import io.realm.RealmObject;

public class ReverseType extends RealmObject {
    private LinkTypes links;

    public LinkTypes getLinks() { return links; }

    public void setLinks(LinkTypes links) { this.links = links; }


}

File: src/services/schema-export/tests/models/all/cs/ReverseType.cs
// Please note : [Backlink] properties and default values are not represented
// in the schema and thus will not be part of the generated models

using System;
using System.Collections.Generic;
using Realms;

namespace MyProject.Models
{
    public class ReverseType : RealmObject
    {
        [MapTo("links")]
        public LinkTypes Links { get; set; }
    }
}
